api/proto/attest/attest.proto

```diff
@@ -18,7 +18,7 @@ message ZAttestNonceReq {
   string deviceUuid = 1;  //UUID assigned by the controller during register
```

Comment:
I am confused about why do we keep including deviceUuid in all these messages. @nordmark @cshari-zededa can you please explain the rationale here?

Reply:
I added this for source identification of the message, in case the message is parsed by a service which does not know about the envelope content.

Reply:
But this feels like an extra burden on both the cloud and the EVE. Should we really care about this right now or can we drop it?

Reply:
Removed UUID from the client messages.